index.html

```diff
@@ -0,0 +1,155 @@
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup"
+        aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse page-padding d-md-flex justify-content-between" id="navbarNavAltMarkup">
+        <a href="#home"><img class="d-none d-md-flex logo flex-start" alt="logo" src="content/images/next-door.svg"></a>
+        <div class="navbar-nav">
+          <a class="nav-item nav-link active me-md-4 text-size-menu" href="#home">Home <span
+              class="sr-only">(current)</span></a>
+          <a class="nav-item nav-link me-md-4 text-size-menu" href="#program">Program</a>
+          <a class="nav-item nav-link me-md-4 text-size-menu" href="/about.html">About</a>
+          <a href="/about.html#about-info" class="d-none nav-item text-decoration-none p-1 d-md-flex text-size-menu logo-name">What's
+            Next</a>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <main class="home-page d-md-flex flex-column page-padding pt-md-5 pb-md-5" id="home">
+    <h3 class="color-primary font-secondary h3 first-page-heading1 weight pt-md-5">"Hello! Sharing Word"</h3>
+    <h1 class="color-primary font-primary display-2 first-page-heading2 weight">LET's SEE WHAT IS NEXT</h1>
+    <p class="home-page-paragraph display-8 secondary border border-white p-2 font-secondary mb-4 mt-4">A joyful
+      celebration and
+      entertaining event with the best
+      and most famous actors, We come
+      together to see the best products that we are going to see in the film market in the near future.</p>
+    <h3 class="font-secondary weight">2022.10.15(THU) ~ 16(FRI)</h3>
+    <p class="font-secondary pb-5 mb-0">@ Sheikh Mohammad bin Rashid Blvd-Downtown Dubai-United Arab Emirates</p>
+  </main>
+  <section class="main-program pt-5 d-flex flex-column align-items-center" id="program">
+    <h2 class="color-thirthary about-page-header text-center h5 mb-3">Main Program</h2>
+    <div class="header-border mb-3">
+    </div>
+    <div class="p-4 d-md-flex pb-md-5">
+      <div
+        class="main-programs d-flex align-items-center row mt-2 col-md flex-md-column align-items-center pt-md-2 me-md-3 ms-md-3">
+        <img src="content/images/speech-svg.svg" class="victor col-2 justify-content-center col-md-4 p-md-0"
+          alt="victor">
+        <h5 class="color-primary col-3 col-offset-2 h5 p-md-0 col-md-3 text-md-center mb-md-0">Lecture</h5>
+        <p class="color-thirthary col-7 font-secondary col-md p-md-0 text-md-center">Listen to the speakers from various
+          companies and organizations
+          around the world about
+          the movies that will
+          realize in 2022.</p>
+      </div>
+      <div
+        class="main-programs d-flex align-items-center row mt-2 col-md d-md-flex flex-md-column align-items-center pt-md-2 me-md-3">
```

Comment:
- Desktop version:
- [ ] Kindly make sure that `Tea-Talk` is aligned with that of the other cards displayed. Also, follow the spacing between text as per the design.
- [ ] Kindly make sure to add the hover effect on all the cards as per the design.
- [ ] Kindly add button interaction (hover) for the button below cards.
- Your design:
<img width="1123" alt="Screenshot 2022-04-21 at 7 28 51 AM" src="https://user-images.githubusercontent.com/70269507/164374800-b6d2e25b-4161-484a-9e87-9d446cafc7b5.png">
- Original design: 
<img width="871" alt="Screenshot 2022-04-21 at 7 58 23 AM" src="https://user-images.githubusercontent.com/70269507/164375395-f79241a2-6e39-4acc-b7d9-2eceec8187c1.png">

- [ ] Mobile version:
- [ ] Kindly Follow the spacing, height and width for the cards. Make sure that all the cards are aligned as per the design.
- Your design:
<img width="496" alt="Screenshot 2022-04-21 at 7 35 00 AM" src="https://user-images.githubusercontent.com/70269507/164374809-0da41604-b554-467f-bbdb-77110c4a4a5e.png">
- Original design:
<img width="406" alt="Screenshot 2022-04-21 at 7 57 58 AM" src="https://user-images.githubusercontent.com/70269507/164375575-4872a464-3955-4b0c-8c55-b5f9ea7848ae.png">